### PR TITLE
Borer Event no longer forces people into borers

### PR DESCRIPTION
**Pull request by Super3222**
### Intent of your Pull Request

Borer event now polls ghosts before spawning the borers instead of gathering ghosts that have BE_BORER and forcing them into borers.

#### Changelog

:cl:
rscadd: Borer event no longer forces people into borers.
/:cl:

### CODE REVIEW

**Comment by Jc1469** [comment]
So to confirm, this waits before spawning the borer too?

**Comment by Super3222** [comment]
> So to confirm, this waits before spawning the borer too?

yes

**Comment by ShadowDeath6** [comment]
@super3222 u ever gonna fix the borer event polling fam?

**Comment by Super3222** [comment]
@ShadowDeath6 i'll have the pr up by next saturday

**Comment by Mrcreepssss** [comment]
YES it's so annoying when this shit happens the moment before you get rezzed.

File: code/modules/mob/living/simple_animal/borer/borer_event.dm [hunk 1]
```diff
@@ -36,9 +36,9 @@
 		if(H.stat != DEAD)
 			total_humans++
 
+	var/list/mob/dead/observer/candidates = pollCandidates("Do you want to play as a borer?", ROLE_BORER, null, 200)
 
-	var/list/candidates = get_candidates("borer", null, ROLE_BORER)
-
+	sleep(210)
```
Comment (Cruix) [inline code/modules/mob/living/simple_animal/borer/borer_event.dm]: pollCandidates() sleeps by itself, you don't need this.

**Pull request merged**